src/selectors/per-thread/composed.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I had to create this extra file of thread selectors because I needed the result of other (stack sample / markers) selectors.